e2e/system_test.go

```diff
@@ -1373,13 +1373,10 @@ func TestWithdrawals(t *testing.T) {
 	header, err = l2Sync.HeaderByNumber(ctx, new(big.Int).SetUint64(blockNumber))
 	require.Nil(t, err)
 
-	var nextHeader *types.Header = nil
-	if sys.RollupConfig.IsBlue(header.Time) {
-		ctx, cancel = context.WithTimeout(context.Background(), 1*time.Second)
-		nextHeader, err = l2Sync.HeaderByNumber(ctx, new(big.Int).SetUint64(blockNumber+1))
-		defer cancel()
-		require.Nil(t, err)
-	}
+	ctx, cancel = context.WithTimeout(context.Background(), 1*time.Second)
+	nextHeader, err := l2Sync.HeaderByNumber(ctx, new(big.Int).SetUint64(blockNumber+1))
+	defer cancel()
```

Comment:
Seeing above code it would be more consistent doing like below.

```suggestion
	nextHeader, err := l2Sync.HeaderByNumber(ctx, new(big.Int).SetUint64(blockNumber+1))
	defer cancel()
```

Reply:
What's the difference?

Reply:
Sorry for confusion. I mean `defer cancle()` should come first!

Reply:
Applied it. Please review again!